src/main/resources/application.properties

```diff
@@ -5,4 +5,6 @@ logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
 #TODO if applicable, edit datasource properties (CAUTION: do not push changes to version control)
 spring.datasource.url=jdbc:mysql://localhost:3306/switchboard?useSSL=false&serverTimezone=UTC
 spring.datasource.username=root
-spring.datasource.password=root
\ No newline at end of file
+spring.datasource.password=root
+#spring.h2.console.enabled=true
+#spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
Can you remove the commented out properties?

Reply:
resolved